tests/expected-matlab/+gtsam/Point2.m

```diff
@@ -10,6 +10,7 @@
 %argUChar(unsigned char a) : returns void
 %dim() : returns int
 %eigenArguments(Vector v, Matrix m) : returns void
+%pickle() : returns void
```

Comment:
This should not be here

Reply:
What part of the wrapper is generating this?